### PR TITLE
add DSF Core and Custom Device to automated build on NIJenkins

### PR DESCRIPTION
**Pull request by dbendele**
This PR automates and standardizes the build for DSF Core and DSF Custom Device.

### CODE REVIEW

File: control_dsf_cd [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+Description: Provides support for the NI Data Sharing Framework Custom Device for NI VeriStand {veristand_version}.
+XB-Eula: eula-ni-standard
+Priority: standard
+Homepage: http://www.ni.com
+XB-LanguageSupport: en
+XB-UserVisible: yes
+Section: Add-Ons
+XB-DisplayVersion: {display_version}
+XB-DisplayName: NI Data Sharing Framework Custom Device for NI VeriStand {veristand_version}
+Depends: {ni-veristand-{labview_version}} (>= {labview_short_version}.0.0)
```
Comment (buckd) [inline control_dsf_cd:15]: `control` files should have an empty line at the end of the file per convention.

Comment (dbendele) [reply control_dsf_cd]: commit 8a55ea6355d354faca6fe261488c5c333438f493

File: .gitignore [hunk 2]
```diff
@@ -11,3 +11,6 @@
 # Metadata
 *.aliases
 *.lvlps
+VeriStand/Data Sharing Custom Device/Built/Custom Device/pharlap/x86/DSF Engine Pharlap.llb
+VeriStand/Data Sharing Custom Device/Source/DSF Core.lvlibp
+Built/Custom Device/pharlap/x86/DSF Engine Pharlap.llb
```
[the remark this answers is not in the document]
Comment (dbendele) [reply .gitignore]: commit 8257c37af5cf63a4493c2ecd2aa2ebcece7d7150

**Comment by Karl-G1** [comment]
Yes. All build nodes have the latest LV memory management build in vi.lib. The lvmm tag can be removed.

File: Jenkinsfile [hunk 3]
```diff
@@ -2,7 +2,7 @@
 //Leave the above line alone.  It identifies this as a groovy script.
 @Library('vs-build-tools') _
 
-def lvVersions = ['2017', '2018', '2019']
+def lvVersions = ['2017', '2018', '2019', '2020']
 
 ni.vsbuild.PipelineExecutor.execute(this, 'lvmm', lvVersions)
```
[the remark this answers is not in the document]
Comment (Karl-G1) [reply Jenkinsfile:7]: Yes. All build nodes have the latest LV memory management build in vi.lib. The lvmm tag can be removed.

[the remark this answers is not in the document]
Comment (dbendele) [reply Jenkinsfile]: 2856e1e96974bbdb099ebe16423aa579c9dcec94

**Comment by dbendele** [comment]
Work in Progress
- [ ] Core Build for Linux x64
- [ ] Custom Device Build for Linux x64

File: control_dsf_cd [hunk 4]
```diff
@@ -1,14 +1,15 @@
-Package: ni-data-sharing-framework-core-support-{veristand_version}-windows
+Package: ni-data-sharing-framework-custom-device-support-veristand-{veristand_version}
 Version: {nipkg_version}
 Architecture: windows_all
```
[the remark this answers is not in the document]
Comment (dbendele) [reply control_dsf_cd]: 9456d2b4901f01efbe12bdcf61f2fd09ee6b1fc9

**Pull request merged**